clouddriver-dcos/src/main/groovy/com/netflix/spinnaker/clouddriver/dcos/model/DcosInstance.groovy

```diff
@@ -44,7 +46,7 @@ class DcosInstance implements Instance, Serializable {
 
     // TODO Instance interfaces says this is the availability zone. Not sure we have this concept - we can only get the host.
     // Should this be our concept of region? Kubernetes uses namespace here.
```

Comment:
We are using the safeCombinedGroup here (not sure if we still need to), but I would say this still applies.